pkg/virt-launcher/virtwrap/network/podinterface_test.go

```diff
@@ -201,8 +214,8 @@ var _ = Describe("Pod Network", func() {
 		mockNetwork.EXPECT().DisableTXOffloadChecksum(bridgeTest.Name).Return(nil)
 
 		// For masquerade tests
-		mockNetwork.EXPECT().LinkByName(primaryPodInterfaceName).Return(dummy, nil)
-		mockNetwork.EXPECT().AddrList(dummy, netlink.FAMILY_ALL).Return(addrList, nil)
+		mockNetwork.EXPECT().LinkByName(originalPodInterfaceName).Return(primaryPodInterfaceBeforeNameChange, nil)
```

Comment:
This line is causing a sorts of errors. No matter what I change the Expected result to, it's wrong.

Leaving it as `dummy` gives the error:
```
    Unexpected call to *network.MockNetworkHandler.AddrList([0xc0016daa20 2]) at pkg/virt-launcher/virtwrap/network/generated_mock_common.go:49 because:
    Expected call at pkg/virt-launcher/virtwrap/network/generated_mock_common.go:56 doesn't match the argument at index 0.
    Got: &{{1 1410 0   0 0 0 0 <nil>  <nil> 0 <nil>  <nil> unknown 0 0 0 0 0 [] 0 <nil>}}
    Want: is equal to &{{0 0 0 eth0  0 0 0 0 <nil>  <nil> 0 <nil>  <nil> unknown 0 0 0 0 0 [] 0 <nil>} }
    Expected call at pkg/virt-launcher/virtwrap/network/generated_mock_common.go:56 doesn't match the argument at index 1.
    Got: 2
    Want: is equal to 0
    Expected call at pkg/virt-launcher/virtwrap/network/generated_mock_common.go:56 doesn't match the argument at index 0.
    Got: &{{1 1410 0   0 0 0 0 <nil>  <nil> 0 <nil>  <nil> unknown 0 0 0 0 0 [] 0 <nil>}}
    Want: is equal to &{{0 0 0 eth0  0 0 0 0 <nil>  <nil> 0 <nil>  <nil> unknown 0 0 0 0 0 [] 0 <nil>} }
```

You'll see the errors in the gate now that I checked in the change.  WDYT?